src/main/java/org/project36/qualopt/domain/Invitation.java

```diff
@@ -0,0 +1,43 @@
+package org.project36.qualopt.domain;
+
+import java.io.Serializable;
+
+import org.project36.qualopt.domain.Study;
+/**
+ * The invitation class is a domain class that is NOT MAPPED
+ * and is used to pass a study object along with the delay to
+ * the resource method.
+ */
+public class Invitation implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private Study study;
```

Comment:
According to Ewan's Java rules, you should add underscores in front of the private fields?

Reply:
I disagree to this one :( 
The existing code base doesn't use the underscore convention, so any code added on top of it probably shouldn't use it either.

Reply:
that's more of a convention isn't it? Plus this project isn't following that, so I didn't put the underscore in